README.markdown

```diff
@@ -8,6 +8,9 @@ Database Cleaner is a set of gems containing strategies for cleaning your databa
 The original use case was to ensure a clean state during tests.
 Each strategy is a small amount of code but is code that is usually needed in any ruby app that is testing with a database.
 
+**NB** Please note that this README is not of the gem's latest version, you may
+find some unexpected errors if you go by it. Find the current version README and docs here: [most recent docs](https://github.com/DatabaseCleaner/database_cleaner/tree/v1.7.0)
```

Comment:
Rather than unexpected errors, its better to say something like `you will find inconsistencies compared to 1.x` they're not errors, they just don't apply to 1.7. You could link to the read me directly here, rather than the root.